src/software/ai/passing/cost_function.h

```diff
@@ -10,36 +10,32 @@
 #include "software/world/team.h"
 #include "software/world/world.h"
 
-// The types of passes we can perform
-MAKE_ENUM(PassType,
```

Comment:
why don't we need this?

Reply:
the pass generator just returns the best pass based on the world, and its up to the receiver to receive and dribble
 
 we didn't use the PassType anywhere before FWIW, it was always configured to  `ONE-TOUCH`